<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmInformacionFarmaco</class>
 <widget class="QDialog" name="FrmInformacionFarmaco">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Informaci贸n medicamento</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icons/PNG/Maya.png</normaloff>:/Icons/PNG/Maya.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nombre:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Receta:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Financiado:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6" colspan="4">
    <widget class="QLineEdit" name="txtLaboratorio">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="txtCodigoNacional">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QLineEdit" name="txtCaja">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Caja:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QDateEdit" name="txtFechaAlta">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>C贸digo nacional:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Fecha Alta:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Nombre ATC:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="5" colspan="2">
    <widget class="QLabel" name="lblfechabaja">
     <property name="text">
      <string>Fecha Baja: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="QLineEdit" name="txtPVC">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="5" colspan="6">
    <widget class="QLabel" name="label_16">
     <property name="minimumSize">
      <size>
       <width>512</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255)</string>
     </property>
     <property name="text">
      <string>Imagenes Medicamento</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="8">
    <widget class="QLineEdit" name="txtDosificacion">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" rowspan="2" colspan="5">
    <widget class="QTableWidget" name="listaIndicaciones">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="8">
    <widget class="QLineEdit" name="txtTipoMedicamento">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Tratamiento larga duraci贸n:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Dosificaci贸n:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="8">
    <widget class="QLineEdit" name="txtNombre">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="5" column="9">
    <widget class="QLineEdit" name="txtPVP">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>92</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="3" colspan="7">
    <widget class="QLineEdit" name="txtFinanciado">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>313</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="8">
    <widget class="QLineEdit" name="txtNombreATC">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="9" column="7">
    <widget class="QDateEdit" name="txtFechaBaja">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="3" colspan="7">
    <widget class="QLineEdit" name="txtReceta">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="11" column="11" colspan="2">
    <widget class="QTableWidget" name="listaInteracciones">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="5" column="8">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Precio:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Tipo Med:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Laboratorio:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="4" colspan="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Precio laboratorio:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="5">
    <widget class="QLabel" name="label_15">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255)</string>
     </property>
     <property name="text">
      <string>Indicaciones</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="11" colspan="2">
    <widget class="QLabel" name="label_17">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255)</string>
     </property>
     <property name="text">
      <string>Posibles Interacciones</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="7">
    <widget class="QLineEdit" name="txtTLD">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 127);
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="10" rowspan="4" colspan="3">
    <widget class="QWebView" name="webLogoLaboratorio" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>175</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="url" stdset="0">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="12" column="11" colspan="2">
    <widget class="QPushButton" name="btnAceptar">
     <property name="text">
      <string>Aceptar</string>
     </property>
     <property name="icon">
      <iconset resource="../maya.qrc">
       <normaloff>:/Icons/PNG/OK.png</normaloff>:/Icons/PNG/OK.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="11" column="5" rowspan="2">
    <widget class="QWebView" name="webimagen1" native="true">
     <property name="url" stdset="0">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="11" column="6" rowspan="2" colspan="2">
    <widget class="QWebView" name="webimagen2" native="true">
     <property name="minimumSize">
      <size>
       <width>164</width>
       <height>0</height>
      </size>
     </property>
     <property name="url" stdset="0">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="11" column="8" rowspan="2" colspan="2">
    <widget class="QWebView" name="webimagen3" native="true">
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>0</height>
      </size>
     </property>
     <property name="url" stdset="0">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="9" column="8" colspan="2">
    <widget class="QLabel" name="lblDadodeBaja">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>MEDICAMENTO DADO DE BAJA</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../maya.qrc"/>
 </resources>
 <connections/>
</ui>
